dlt/destinations/impl/duckdb/configuration.py

```diff
@@ -95,28 +96,25 @@ class DuckDbCredentials(DuckDbBaseCredentials):
 
     __config_gen_annotations__: ClassVar[List[str]] = []
 
-    def on_resolved(self) -> None:
```

Comment:
What is the problem here? `on_resolved` is not called in new model because we allow for partial? but at some point we must make sure that configuration is complete. pls see my other comments

Reply:
Ok my fix was a problem for dbt runner so I went back to the original `on_resolved` method, but added this https://github.com/dlt-hub/dlt/blob/sthor%2Fparametrized-destination/dlt/destinations/impl/duckdb/configuration.py#L99-L104 so it doesn't trigger right when the instance is created